manifests/init.pp

```diff
@@ -21,7 +128,7 @@
   $config_dir_group    = 'root',
   $config_dir_mode     = '0755',
   $config_ensure       = 'file',
-  $config_path         = '/etc/eyaml/config.yaml',
```

Comment:
Config_dir is pointing to $::settings::confdir so it didn't make sense that the config_path was different right?